project/settings/base.py

```diff
@@ -255,6 +255,7 @@ def __mod__(self, other):
                 # 'social_django.context_processors.backends',
                 "social_django.context_processors.login_redirect",
                 "django.template.context_processors.static",
+                "django.template.context_processors.media",
```

Comment:
This should be no longer relevant now that you're passing in the URLs via a context variable.